### DISCUSSION TITLE
Recent update broke biometric unlock feature in Firefox browser add-on (MacOS)?

### DISCUSSION THREAD

**Discussion by jaydee73** [dani-garcia/vaultwarden#3829] [Q&A] [answered]
I am using Vaultwarden (docker image), Mac OS and Firefox with Bitwarden Browser add-on and Bitwarden Mac OS Desktop client. All with biometric unlock. Everything was working fine until today.. ;-)

Now, when I open Firefox and click on the Bitwarden browser add-on icon, I am asked to unlock with fingerprint. This still is the same behaviour as yesterday => The macos biometric system pop-up window comes up and asks for my fingerprint. But the Bitwarden browser add-on then seems not to recognize the biometric unlock. It stucks with "waiting for confirmation from the Desktop" (I'm not sure if this is the right translation, in German it says "Warte auf Bestätigung vom Desktop"). I can still unlock the vault with my Master password in the browser add-on, but biometrics do not work anymore.

They were some updates yesterday/today, so I am actually not sure which update is responsible for this behaviour. I updated my docker vaultwarden container today, updated Firefox to 117 yesterday and also updated my Bitwarden Desktop Client today. 

Side note: When I am using Safari with the Safari-Bitwarden-Browser-Add-on, biometric unlock still works as expected.

I de- and re-installed Firefox => no change. I de- and re-installed the Bitwarden Firefox add-on => no change.

Can anyone probably reproduce this in his own mac os environment? I'd like to have another confirmation before I'll raise an issue here.

Thanks,
Stefan

**Comment by BlackDex** [comment, marked as answer]
This has nothing to do with the server backend.
It doesn't request anything in regards to the biometric unlock.

It probably is related to https://github.com/bitwarden/clients/issues/6156

**Reply by jaydee73** [reply]
Yes, #6156 exactly describes it. Thanks. I will continue to monitor the thread until a solution is posted there.
